File: FoodBooking.Data/Entities/Restaurant.cs
namespace FoodBooking.Data.Entities
{
    public class Restaurant
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public Guid? ImageId { get; set; }
        public DateTime CreateDate { get; set; }
        public string LinkCrawl { get; set; }
        public int SourceCrawlId { get; set; }
        public Image Image { get; set; }
        public ICollection<Product> Products { get; set; }
    }
}

File: FoodBooking.UI/src/app/models/restaurant.ts
import { Guid } from "guid-typescript";
import { IImage } from "./image-model";
import { IProduct } from "./product-model";

export interface IRestaurant
{
    id:Guid,
    name:string,
    description:string,
    title:string,
    image: IImage,
    createDate: Date,
    address:string,
    linkCrawl: string,
    sourceCrawlId: number,
    brand: string,
    products: IProduct[]

}

export interface IRestaurantsReponse
{
    restaurants:IRestaurant[],
    page:number,
    totalRow:number,
    totalPage:number
}


export enum ESourceCrawl {
    None,
    Shoppe,
    Grab
}